### DISCUSSION TITLE
Brim Ears from superslicer please

### DISCUSSION THREAD

**Discussion by leezus-nz** [OrcaSlicer/OrcaSlicer#417] [Ideas]
This is the one feature I am missing from superslicer. I'm over manually placing mouse ears since I switched back to Prussia slicer. Now I'm here, it's would make my day to get some brim ears in your fork.

Thanks for the awesome work!

**Comment by MartianDrummer** [comment]
This is an option that I use (maybe more than I should) and would also LOVE to see it in Orca

Thanks so much for your great work!!!
